src/evidently/types.ts

```diff
@@ -0,0 +1,51 @@
+export type EvaluationCallback = (err?: Error, res?: EvaluationResults) => void;
+
+export type ContextType = { [k: string]: any };
+export type PartialEvidentlyConfig = {
+    project: string;
+    endpoint?: URL;
+};
+export type EvidentlyConfig = {
+    project?: string;
+    endpoint: URL;
+};
+export type EvidentlyRequest = {
+    features: string[];
+    entityId?: string;
+    context?: ContextType;
+};
+type VariationValue = {
+    boolValue?: boolean;
+    doubleValue?: number;
+    longValue?: number;
+    stringValue?: string;
+};
+export type EvaluationResult = {
+    feature: string;
+    reason: string;
+    value: VariationValue;
+    variation: string;
+};
+export type EvaluationResults = { [feature: string]: EvaluationResult };
+
+export type APIEvaluationRequest = {
+    entityId: string;
+    evaluationContext?: string;
+    feature: string;
+};
+export type BatchEvaluateFeatureRequest = {
+    requests: APIEvaluationRequest[];
+};
+
+export type APIEvaluationResult = {
```

Comment:
I think we should just copy all the types from the API details and not name them something different. E.g. this should be called `EvaluationResult` based on the specs, not `APIEvaluationResult`: https://docs.aws.amazon.com/cloudwatchevidently/latest/APIReference/API_EvaluationResult.html

I would call the new data types something new, so we don't overload the names.